src/maven/background_peaks_update.cpp

```diff
@@ -422,7 +422,7 @@ void BackgroundPeakUpdate::processSlices(vector<mzSlice*>&slices, string setName
     QTime timer2;
     timer2.start();
     qDebug() << "Building searchable database.";
-    vector<tuple<float, Compound*, Adduct*>> searchableDatabase = prepareCompoundDatabase(this->compounds);
+    vector<CompoundIon> searchableDatabase = prepareCompoundDatabase(this->compounds);
     qDebug() << "Built searchable database of " << searchableDatabase.size() << " entries in " << timer2.elapsed() << " sec.";
```

Comment:
The `CompoundIon` class is a container class that contains the `(Compound*, Adduct*, precursorMz (float))` tuple.